assets/dev/scss/frontend/widgets/image.scss

```
@@ -6,6 +6,11 @@
 
 	a {
 		display: inline-block; //For alignment image with link - Changed to 'inline-block' instead of 'block' to handle with this issue https://github.com/elementor/elementor/issues/5897
+	  	width: var( --img-width ); // To resolve ED-3397
```

There is no need for a comment here. Generally, we do not include ticket IDs in source code.

```suggestion 
		width: var( --img-width );
```

---

D